app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -258,7 +258,22 @@
             }}
           </p>
         </label>
-
+        <div class="medium-9 settings-item settings-item">
+          <label>
+            {{ $t('INBOX_MGMT.HELP_CENTER.LABEL') }}
+          </label>
+          <select v-model="selectedPortalSlug" class="filter__question">
+            <option value="" disabled>
+              {{ $t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER') }}
+            </option>
```

Comment:
At the moment, it's not possible to reset the value once set, we could do one of either things. Do we want to be able to reset it?

#### Add an option called "None" and hide the placeholder option

```suggestion
            <option value="" disabled hidden>
              {{ $t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER') }}
            </option>
            <option value="">
              {{ $t('INBOX_MGMT.HELP_CENTER.NONE') }}
            </option>
```

#### Use the placeholder as the de-selected value

```suggestion
            <option value="">
              {{ $t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER') }}
            </option>
```


#### Add a reset button

Reply:
@scmmishra  Having "None" value now to reset it, handling 'null' string in the backend, we will remove that line after fixing https://linear.app/chatwoot/issue/CW-1567/null-value-passed-as-null-string-to-backend